File: TP2DPBO2526C1/Python/kota.py
from negara import Negara


class Kota(Negara):
    def __init__(self, nama_benua, luas_benua, zona_waktu_utama, nama_negara, bahasa_resmi, mata_uang,
                 nama_kota, jumlah_penduduk, ikon_kota):
        super().__init__(nama_benua, luas_benua, zona_waktu_utama, nama_negara, bahasa_resmi, mata_uang)
        self.nama_kota = nama_kota
        self.jumlah_penduduk = jumlah_penduduk
        self.ikon_kota = ikon_kota

    def print_row(self):
        print(f"{self.nama_benua:<18} {self.luas_benua:<15} {self.zona_waktu_utama:<15} "
              f"{self.nama_negara:<15} {self.bahasa_resmi:<15} {self.mata_uang:<15} "
              f"{self.nama_kota:<18} {self.jumlah_penduduk:<18} {self.ikon_kota:<20}")

File: TP2DPBO2526C1/Java/Kota.java
class Kota extends Negara {
    private String nama_kota;
    private long jumlah_penduduk;
    private String ikon_kota;

    public Kota(String nb, long lb, String zvu, String nn, String br, String mu, String nk, long jp, String ik) {
        super(nb, lb, zvu, nn, br, mu);
        this.nama_kota = nk;
        this.jumlah_penduduk = jp;
        this.ikon_kota = ik;
    }

    public void printRow() {
        System.out.printf("%-18s %-15d %-15s %-15s %-15s %-15s %-18s %-18d %-20s\n", 
            nama_benua, luas_benua, zona_waktu_utama, nama_negara, bahasa_resmi, mata_uang, nama_kota, jumlah_penduduk, ikon_kota);
    }
}
